zinit/upgraded.yaml

```diff
@@ -1,4 +1,4 @@
-exec: upgraded
+exec: upgraded -root /var/cache/modules/upgrade
```

Comment:
It seems upgraded was using a location outside the cache disk which means it used the tmpfs to download the files. Don't think this was intended

Reply:
upgraded doesn't download anything, it just uses the flist module. and does the copy from there into the rootfs. But anyhow your change are good